docs/Tables.md

```diff
@@ -63,45 +69,102 @@ pdf.ln()
 with pdf.table(text_align=("CENTER", "CENTER", "RIGHT", "LEFT")) as table:
     ...
 ```
+
 Result:
 
 ![](table_align.jpg)
 
+## Setting cell padding
+
+_New in [:octicons-tag-24: 2.7.5](https://github.com/PyFPDF/fpdf2/blob/master/CHANGELOG.md)_
```

Comment:
There's always "just one other thing"... 😉 

This will actually go into 2.7.6. (also below), since 2.7.5 has been out for a while now.

Reply:
haha indeed.